cli/build.rs

```diff
@@ -112,6 +117,10 @@ fn main() {
     "cargo:rustc-env=DENO_WEB_LIB_PATH={}",
     deno_web::get_declaration().display()
   );
+  println!(
+    "cargo:rustc-env=DENO_FETCH_LIB_PATH={}",
+    deno_fetch::get_declaration().display()
+  );
```

Comment:
This method of passing the declaration file is not so elegant. I hope we can find a better solution in the future.

Reply:
I agree, if it's build dependencies there's surely a way to include it without using `env!()` macro